libs/assets/src/lib/asset-option.tsx

```diff
@@ -16,39 +15,74 @@ export const Balance = ({
   balance?: string;
   symbol: string;
 }) => {
-  const t = useT();
   return balance ? (
-    <div className="mt-1 font-alpha" data-testid="asset-balance">
+    <div className="font-alpha text-xs" data-testid="asset-balance">
       {balance} {symbol}
     </div>
   ) : (
-    <div className="text-vega-orange-500" data-testid="asset-balance">
-      {t('Fetching balance…')}
+    <div className="text-muted text-xs" data-testid="asset-balance">
+      -
     </div>
   );
 };
 
 export const AssetOption = ({ asset, balance }: AssetOptionProps) => {
+  const chainId = asset.source.__typename === 'ERC20' && asset.source.chainId;
+  const assetSource =
+    asset.source.__typename === 'ERC20' && asset.source.contractAddress;
+  let assetLogo = 'https://icon.vega.xyz/missing.svg';
+  let chainLogo = 'https://icon.vega.xyz/missing.svg';
+  if (chainId && assetSource) {
+    assetLogo = `https://icon.vega.xyz/chain/${chainId}/asset/${assetSource}/logo.svg`;
+    chainLogo = `https://icon.vega.xyz/chain/${chainId}/logo.svg`;
+  }
```

Comment:
**question (non-blocking)**: Why do this 'manually' rather than use the `Emblem` components? It's not a problem but I wonder if there's something I should improve about those components.